src/backend/dx12/src/command.rs

```diff
@@ -389,63 +389,66 @@ pub struct CommandBuffer {
     pool_shared: Arc<PoolShared>,
     begin_flags: com::CommandBufferFlags,
 
-    // Cache renderpasses for graphics operations
+    /// Cache renderpasses for graphics operations
```

Comment:
looks like something went totally wrong with the rebase? These changes aren't supposed to be here

Reply:
Fixed. :+1: